src/test/java/uk/gov/hmcts/reform/dataextractor/DataExtractorOutputTest.java

```diff
@@ -18,52 +24,57 @@ public void setup() {
 
     @Test
     public void whenDefaultOutputRequested_thenJsonLinesReturned() {
-        assertEquals(DataExtractorApplication.Output.JSON_LINES, DataExtractorApplication.Output.defaultOutput());
+        assertEquals(Output.JSON_LINES, Output.defaultOutput());
```

Comment:
PMD forces me to add an assertion message to each of these lol. But honestly, I don't think they are that useful. So not necessary for me.

Reply:
Not really sure why PMD is not complaining here, i also think it's not necessary the message.. I'll keep it as is it